pkg/cloud/rgraph/exec/executor.go

```diff
@@ -57,6 +56,19 @@ func DryRunOption(dryRun bool) Option {
 	return func(c *ExecutorConfig) { c.DryRun = dryRun }
 }
 
+// TimeoutOption sets timeout for executor Run function.
+// This option can be used with parallel executor only.
```

Comment:
Can we have a follow up to make this work with the SerialExecutor 

Actually, what is actually required to make it work? Isn't this simple change?